services/api/src/models/Quests.test.ts

```diff
@@ -10,7 +10,7 @@ import {
 describe('quest', () => {
   describe('searchQuests', () => {
     it('returns an empty array if no results', (done: DoneFn) => {
-      testingDBWithState([q.basic, q.expansion])
+      testingDBWithState([q.basic, q.horror, q.future])
```

Comment:
Maybe this array should be a `const` value in the root of the `searchQuests` section...

Reply:
Done